tree/ntuple/v7/inc/ROOT/RFieldVisitor.hxx

```diff
@@ -34,8 +34,26 @@ namespace Detail {
 \ingroup NTuple
 \brief Abstract base class for classes implementing the visitor design pattern.
 
-RFieldVisitor::VisitField() is invoked by RFieldBase::AcceptVisitor(). VisitField() is inherited for instance
-by the RPrintSchemaVisitor class. The RFieldBase class and classes which inherit from it will be visited.
+RFieldVisitor::VisitField() is invoked by RFieldBase::AcceptVisitor().
+For an example use case, see Internal::RPrintSchemaVisitor.
```

Comment:
Is this sentence needed with the below example? Perhaps move it below the example and phrase it as "For another example..."